README.md

```diff
@@ -91,316 +73,114 @@ Based on the [Extension](https://hl7.org/fhir/R4B/extensibility.html#Extension)
+Based on [FHIR AllergyIntolerance](https://hl7.org/fhir/R4B/allergyintolerance.html), the following properties may be used, while additional properties are ignored by the Engage-HF system.
 
 |Property|Type|Values|Comments|
 |-|-|-|-|
-|dateOfBirth|Date|-|To be used for verification purposes.|
-|dateOfEnrollment|Date|-|The date when the invitation code was used to create this user.|
-|invitationCode|string|-|The invitationCode to be used when logging in to the app for the first time.|
-|language|optional string|e.g. "en"|Following IETF BCP-47 / [FHIR ValueSet languages](https://hl7.org/fhir/R4B/valueset-languages.html).|
-|messagesSettings|-|-|See properties below.|
-|messagesSettings>dailyRemindersAreActive|boolean|true, false|Decides whether to send out daily reminder messages for this user.|
-|messagesSettings>textNotificationsAreActive|boolean|true, false|Decides whether to send text notifications for this user.|
-|messagesSettings>medicationRemindersAreActive|boolean|true, false|Decides whether to send medication reminder messages for this user.|
-|timeZone|string|e.g. "America/Los_Angeles"|The value needs to correspond to an identifier from [TZDB](https://nodatime.org/TimeZones). It must not be an offset to UTC/GMT, since that wouldn't work well with daylight-savings (even if there is no daylight-savings time at that location). Also, don't use common abbreviations like PST, PDT, CEST, etc (they may be ambiguous, e.g. CST). If the timeZone is unknown, then "America/Los_Angeles" should be used.|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|type|optional [allergyIntoleranceType](https://hl7.org/fhir/R4B/valueset-allergy-intolerance-type.html)|e.g. "allergy", "intolerance"|In addition to the FHIR defined value set, we also use "financial" - these values shall not be exposed to EHR systems.|
+|code|CodableContent|e.g. "{"coding":[{"system":"https://hl7.org/fhir/R4B/valueset-allergyintolerance-code.html","code":"293963004","display":"Cardioselective beta-blocker allergy"}],"text":"Cardioselective beta-blocker allergy"}"|Uses either [AllergyIntoleranceCode](https://hl7.org/fhir/R4B/valueset-allergyintolerance-code.html), `medicationId` as used in /medications/$medicationId$ and/or `medicationId` as used in /medicationClasses/$medicationClassId$.|
+|patient|string|-|`userId` as used in /users/$userId$ and related collections.|
 
-### /users/$userId$/appointments/$appointmentId$
+We use RxNorm codes to identify contraindications using the following rules:
+
+|Class|Medication|Brand Name(s)|Dose1|Dose2|Dose3|Dose4|Dose5|Dose6|If allergy, not eligible for|If allergy with reaction type angioedema not eligible for|If intolerance not eligible for|If intolerance default to|If financial, not eligible for|
```

Comment:
done, since this is only relevant for contraindications anyways, there is no necessity to refer to the config file there. I added it to the medications section instead.

Reply:
Sounds good; thank you!